src/App/Header/UserMenu/Menu.tsx

```diff
@@ -11,15 +10,13 @@ type Props = {
   signOut(): void;
   user: AuthenticatedUser;
 };
-export default function Menu({ classes = "", user, signOut }: Props) {
+export default function Menu({ user, signOut }: Props) {
   return (
-    <Popover.Panel
-      className={`${classes} shadow-xl bg-gray-l6 dark:bg-blue-d6 w-max rounded overflow-hidden`}
-    >
+    <>
+      <hr className="my-3 text-navy-l1" />
       <p className="text-sm p-3 bg-blue-l4">
         Welcome, {user.firstName || user.email}!
```

Comment:
The "Welcome" part should probably now be moved to [the top of `NavDropdown` component](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2815/files#diff-1e8583bb89deb5a6e837ff5b2f51a7aaeda45043b30ce5c8a5d09cbb3e7aac00R56), as the message in the middle of the menu looks awkward.

Reply:
Yeah, I struggled with the look/feel for that. Reason for doing it this way was to If it's at the top, it shifts the existing nav menu when comes in. Figured, for now it's cleaner to have a clear separation of navmenu links & user menu links. If Operations folks can drive the UI refinement over the next week or so before launch.